R/prettyOncoplot.R

```diff
@@ -139,1027 +139,1117 @@
-                          keepSampleOrder = FALSE,
-                          highlightHotspots = FALSE,
-                          these_samples_metadata,
-                          metadataColumns,
-                          numericMetadataColumns,
-                          expressionColumns = c(),
-                          numericMetadataMax,
-                          sortByColumns,
-                          arrange_descending = FALSE,
-                          removeNonMutated = FALSE,
-                          minMutationPercent,
-                          mutAlpha = 1,
-                          recycleOncomatrix = FALSE,
-                          splitColumnName,
-                          splitGeneGroups,
-                          showTumorSampleBarcode = FALSE,
-                          groupNames,
-                          hide_annotations,
-                          hide_annotations_tracks = FALSE,
-                          annotate_specific_genes = FALSE,
-                          this_forest_object = NULL,
-                          custom_colours = NULL,
-                          hideTopBarplot = TRUE,
-                          tally_all_mutations = FALSE,
-                          tally_all_mutations_max = 1000,
-                          hideSideBarplot = FALSE,
-                          box_col = NA,
-                          annoAlpha = 1,
-                          legend_direction = "horizontal",
-                          ylim = NULL,
-                          legend_position = "bottom",
-                          legend_row = 3,
-                          legend_col = 3,
-                          metadataBarHeight = 1.5,
-                          metadataBarFontsize = 5,
-                          legendFontSize = 10,
-                          fontSizeGene = 6,
-                          annotation_row = 2,
-                          annotation_col = 1,
-                          verbose = FALSE,
-                          cluster_rows = FALSE,
-                          cluster_cols = FALSE,
-                          cluster_numeric_rows = TRUE,
-                          cluster_numeric_cols = FALSE,
-                          clustering_distance_rows = "binary",
-                          clustering_distance_cols = "binary",
-                          split_rows_kmeans,
-                          split_columns_kmeans,
-                          dry_run = FALSE,
-                          simplify_annotation= FALSE,
-                          stacked = FALSE,
-                          numeric_heatmap_type = "aSHM",
-                          numeric_heatmap_location = "top",
-                          return_inputs = FALSE){
-  
-  if(stacked){
-    if(!simplify_annotation){
-      message("stacked mode is only compatible in combination with simplify = TRUE. Setting this for you.")
-      simplify_annotation = TRUE
+prettyOncoplot = function(
+    maf_df,
+    cnv_df,
+    onco_matrix_path,
```

Comment:
This is now the only reference to `onco_matrix_path` in the whole function. Is it time to remove it and the `recycleOncomatrix` arg?

Reply:
just pushed the update deprecating this argument! ✅

Reply:
passed all checks 😎